src/uct/ib/rc/base/rc_iface.c

```diff
@@ -75,14 +75,14 @@ ucs_config_field_t uct_rc_iface_common_config_table[] = {
    ucs_offsetof(uct_rc_iface_common_config_t, ooo_rw), UCS_CONFIG_TYPE_BOOL},
 #endif
 
-  {"FENCE", "weak",
+  {"FENCE", "none",
```

Comment:
need to enable weak fence by default if pci atomics are used

Reply:
didn't get...
config is parsed before iface is initialized... how default value may be updated?

Reply:
need that in runtime the fence mode will be weak
maybe add new config mode "auto"

Reply:
we were going to add auto mode when adaptive routing detection is implemented.
what about set "weak" by default?

Reply:
we don't want fence at all without pci atomics
because this is runtime decision we have to add auto mode now..

Reply:
ok, added